pkgs/top-level/all-packages.nix

```
@@ -34826,7 +34826,7 @@ with pkgs;
 
   lguf-brightness = callPackage ../misc/lguf-brightness { };
 
-  lilypond = callPackage ../misc/lilypond { guile = guile_1_8; };
+  lilypond = callPackage ../misc/lilypond { guile = guile_2_2; };
```

```suggestion
  lilypond = callPackage ../misc/lilypond { };
```
that is the default guile version